docs/README.md

```diff
@@ -20,10 +20,18 @@ For more info, see [Directory Structure](https://gohugo.io/getting-started/direc
 
 ## Building
 
+Before starting, you must install the [Learn Theme](https://github.com/matcornic/hugo-theme-learn). This can be done by running `git submodule update` within the local repository.
```

Comment:
The theme is linked as a git submodule in `themes/learn` to https://github.com/docksal/hugo-theme-learn, which is a fork of https://github.com/matcornic/hugo-theme-learn.

If changes are needed to the theme, then they would happen in the theme repo first, then re-linked in this repo. This process is somewhat heavy, but it's the only one that would allow for easy updates from upstream. I'm open to suggestion to improve/simplify this flow. We should also add this to the docs here.

I this we should document the recommendation from [here](https://stackoverflow.com/questions/4611512/is-there-a-way-to-make-git-pull-automatically-update-submodules) to configure git to pull submodules automatically:

```
git config --global submodule.recurse true
```